packages/protocol/migrationsConfig.js

```diff
@@ -1,6 +1,11 @@
+const BigNumber = require('bignumber.js')
 const minimist = require('minimist')
 const path = require('path')
 
+// Almost never use exponential notation in toString
+// http://mikemcl.github.io/bignumber.js/#exponential-at
+BigNumber.config({ EXPONENTIAL_AT: 1e9 })
```

Comment:
Why not go even higher?

Reply:
This is the highest the library allows (as per link in comment).

Note that this is the value of the exponent, so it would only start returning exponential notation at 10^1e9, which is a lot.